content/about/faq.md

```diff
@@ -2,48 +2,86 @@
-Yes. Our main goal is to foster FOSS4G, and that means also making all the presentations available even to people that didn’t register in the first place. After the conference, the recorded talks will be published on our YouTube channel - https://www.youtube.com/@FOSS4G  
+Yes, all presentations will be recorded and made publicly available after the
+conference. Our main goal is to foster FOSS4G, and that means also making all
+the presentations available even to people that didn’t register in the first
+place. After the conference, the recorded talks will be published on our
+YouTube channel - https://www.youtube.com/@FOSS4G  
 {{</ faq >}}
 
 {{< faq "Have you considered a hybrid online-face to face event?" >}}
-FOSS4GE 2025 will be an in-person event.  
+Yes, FOSS4G Europe 2025 is organized as an in-person event. However, to
+accommodate a wider audience, we plan to provide online streaming of the
+conference sessions. This approach allows geospatial enthusiasts who cannot
+attend in person to follow the talks and participate virtually.  
 {{</ faq >}}
 
 {{< faq "I am not dedicated to geoinformatics, can I attend the conference?" >}}
-Yes, the conference is open to any public interested in geoinformatics, free software, maps, and communities.  
+Absolutely. The conference welcomes anyone interested in geospatial technologies
+and open-source software. Whether you're a professional from another field, a
+student, or simply curious about geospatial innovations, your participation is
+encouraged.  
 {{</ faq >}}
 
 {{< faq "If I don’t develop or write code, is this conference for me, too?" >}}
-Yes, this event is for all kinds of people interested in sharing experiences about maps, from developers who write code to users of geoinformatics software, people who make decisions, researchers, students, and anyone who wants to learn, share and meet like-minded people.  
+Yes, the conference is designed for a diverse audience, including users,
+educators, and decision-makers. You don't need to be a developer or coder to
+benefit from the sessions and networking opportunities. The event offers
+insights into the application and impact of geospatial technologies across
+various sectors.  
 {{</ faq >}}
 
 {{< faq "What activities will the conference have?" >}}
-The conference will have tracks full of talks, workshops, and community sprint activities. Social programme is very diverse from Geochicas event to Ice-Breaker and Social Bytes.
+The conference will feature a variety of activities, including:
+
+- **Workshops:** Hands-on sessions led by experts to enhance your skills in
+open-source geospatial technologies.
+- **Presentations:** Talks and lectures covering the latest advancements and
+applications in the geospatial field.
+- **Networking Events:** Opportunities to connect with professionals from
+around the world, such as GeoBiz Connect and GeoGals Night.
+- **Social Activities:** Informal gatherings like Meet & Mingle (M&M) and
```

Comment:
@markopuljic If you choose to alter the traditional networking event "codenames" then people might not understand what they are. E.g. from here I understand that there is no Ice-Breaker taking place. And there's no B2B? 

What is the reason why you want to do it?

The reason for no "Gala dinner" in Tartu was that there is nothing _gala_ in t-shirts/shorts/flip-flops and _dinner_ in buffet table. So we ended up with "Social Bytes" without changing the concept which I think reflects the event better.